library/src/main/java/com/chrynan/chords/widget/ChordWidget.kt

```diff
@@ -381,9 +381,10 @@ class ChordWidget : View,
 
         chord?.bars?.forEach { bar ->
             if (bar.fret.number in chart.fretStart..chart.fretEnd && bar.endString.number < chart.stringCount + 1) {
+                val relativeHeight = bar.fret.number - (chart.fretStart - 1)
```

Comment:
This isn't a height value but rather an index, or fret number, so the name of the property seems misleading. `relativeFretNumber` would be more appropriate.

Reply:
Got it; changed.